golang/messages.go

```diff
@@ -152,7 +152,7 @@ const (
 	FailureDomain TransportHeaderName = "fd"
 
 	// ShardKey header value is used by ringpop to deliver calls to a specific tchannel instance.
-	ShardKey TransportHeaderName = "sk"
+	ShardKey TransportHeaderName = "shardKey"
```

Comment:
The shard key transport header key is `sk` according to the protocol docs,
https://github.com/uber/tchannel/blob/master/docs/protocol.md#user-content-transport-headers

Please revert this change.